app/Services/Validation/ApplicationTimelineValidator.php

```diff
@@ -330,6 +330,23 @@ public function validateComplete(JobApplication $application)
 
     /* Step 6 ------------------------------------------------------------------------------------------------------- */
 
+    /**
+     * Affirmation Rules.
+     * @var mixed
+     */
+    public $affirmationRules = [
```

Comment:
Now that these rules have been extracted like this, make sure to reference this in the affirmationValidator function, instead of copy-pasting the rules.